suricata.yaml.in

```diff
@@ -84,6 +84,9 @@ outputs:
       enabled: @e_enable_evelog@
       filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
       filename: eve.json
+      # Enable for multi-threaded eve.json output; output files are suffixed
+      # with an identifier, e.g., eve.json.9.
+      #per-thread: false
```

Comment:
Not really a fan of `per-thread` as the option name here.  In the pcap output we use `multi`, which I'm not a fan of either.  Any other ideas for a name? I'm stuck on simply `threaded`.

Reply:
Thanks ... will change option name to `threaded`